src/command_modules/azure-cli-container/README.rst

```diff
@@ -0,0 +1,186 @@
+Microsoft Azure CLI 'container' Command Module
+==============================================
+
+Commands to manage Azure container instances
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Group
+        az container: Manage Azure Container Instances.
+
+    Commands:
+        create : Create or update container group.
+        delete : Delete a container group.
+        list   : List container groups by resource group.
+        listall: List all container groups.
+        logs   : Tail the log of a container group.
+        show   : Show the details of a container group.
+
+Commands to create an Azure container group
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Command
+        az container create: Create or update container group.
+
+    Arguments
+        --image             [Required]: The container image name.
```

Comment:
Should this have an `-i` alias? That would match the image name arg alias that `az webapp create` supports.